cmd/lima-guestagent/distro_linux.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/reproducible-containers/repro-get/pkg/distro"
+	"github.com/reproducible-containers/repro-get/pkg/distro/alpine"
+	"github.com/reproducible-containers/repro-get/pkg/distro/arch"
+	"github.com/reproducible-containers/repro-get/pkg/distro/debian"
+	"github.com/reproducible-containers/repro-get/pkg/distro/distroutil/detect"
+	"github.com/reproducible-containers/repro-get/pkg/distro/fedora"
+	"github.com/reproducible-containers/repro-get/pkg/distro/none"
+	"github.com/reproducible-containers/repro-get/pkg/distro/ubuntu"
```

Comment:
I'd prefer to just os/exec it

Reply:
Maybe this particular function should be moved from cmd to pkg, upstream ? (in repro-get)